test/parallel/test-event-emitter-get-max-listeners.js

```diff
@@ -27,8 +6,13 @@ var emitter = new EventEmitter();
 
 assert.strictEqual(emitter.getMaxListeners(), EventEmitter.defaultMaxListeners);
 
-emitter.setMaxListeners(0)
-assert.strictEqual(emitter.getMaxListeners(), 0)
+emitter.setMaxListeners(0);
+assert.strictEqual(emitter.getMaxListeners(), 0);
+
+emitter.setMaxListeners(3);
+assert.strictEqual(emitter.getMaxListeners(), 3);
 
-emitter.setMaxListeners(3)
-assert.strictEqual(emitter.getMaxListeners(), 3)
+// https://github.com/iojs/io.js/issues/523 - second call should not throw.
+var recv = {};
+EventEmitter.prototype.on.call(recv, 'event', function() {});
+EventEmitter.prototype.on.call(recv, 'event', function() {});
```

Comment:
No.  The code path isn't reached the first time due to an optimization in how listeners are stored.